frontend/src/lib/components/LemonTextArea/LemonTextArea.tsx

```diff
@@ -63,13 +69,96 @@ interface LemonTextMarkdownProps {
 }
 
 export function LemonTextMarkdown({ value, onChange, ...editAreaProps }: LemonTextMarkdownProps): JSX.Element {
+    const { objectStorageAvailable } = useValues(preflightLogic)
```

Comment:
## object storage disabled

<img width="495" alt="Screenshot 2022-10-13 at 16 53 49" src="https://user-images.githubusercontent.com/984817/195646487-2efbf0aa-09b5-4777-9cb3-954932956eb3.png">

## object storage enabled

<img width="499" alt="Screenshot 2022-10-13 at 16 51 23" src="https://user-images.githubusercontent.com/984817/195646566-c0e6ff39-7407-4802-95b9-65c6258bc2e6.png">

Reply:
Cool! Could we just make it so that the text fits into one line? Two draw a bit too much attention to some extra info. For the drag-and-drop-on case it could just say "Attach images by dragging and dropping them", if unavailable… The message might not be necessary? The question is, in what situations is object storage unavailable (besides not having the `object_storage` service running in local dev)? If this is only when something is misconfigured, we can be pretty verbose; if it's common and totally acceptable, a big "Cannot upload ..." message is going to be annoying.

BTW "Markdown" should be capitalized, it's a proper noun unlike "markup" 😅)

Reply:
got them both to one line and moved object storage note into a tooltip

<img width="498" alt="Screenshot 2022-10-13 at 18 09 17" src="https://user-images.githubusercontent.com/984817/195661983-9e7a8f86-ab41-4128-b290-703fc187340d.png">
<img width="670" alt="Screenshot 2022-10-13 at 18 07 54" src="https://user-images.githubusercontent.com/984817/195661987-ee7e591d-0602-456c-8ccf-6ec9d3d8a9bc.png">